src/pydot/dot_parser.py

```diff
@@ -383,6 +384,29 @@ def push_node_stmt(s: str, loc: int, toks: ParseResults) -> "pydot.core.Node":
     return n
 
 
+class HTML(Token):
+    def __init__(self) -> None:
+        super().__init__()  # type: ignore
```

Comment:
BTW, that linked PR (pyparsing/pyparsing#596) is meant to fix things so we can eventually drop this `type: ignore` comment.